src/cider/exec/nextgen/context/Buffer.h

```diff
@@ -39,6 +39,16 @@ class Buffer {
 
   ~Buffer() { allocator_->deallocate(buffer_, capacity_); }
 
+  void allocateBuffer(int32_t size) {
+    if (capacity_) {
+      buffer_ = allocator_->reallocate(buffer_, capacity_, size);
+      capacity_ = size;
+    } else {
+      buffer_ = allocator_->allocate(size);
+      capacity_ = size;
```

Comment:
Minor: merge & move L45 & L48 in common path.

Reply:
FIX